engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -21,14 +23,23 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  sealed trait ValidateActorMessage
-  case object ValidateWorkflow extends ValidateActorMessage
-
   def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
     Props(new ValidateActor(wdlSource, wdlJson, workflowOptions))
   }
+
+  sealed trait ValidateActorMessage
+  sealed trait ValidationResult extends ValidateActorMessage
+  // This is the reply that this actor sends in case of a succsesful validation. It contains all the resolved / validated components of
+  // the workflow (minus Declarations). The callee should reuse these materialized information.
```

Comment:
Typo in successful. Also this is factually incorrect atm? The actor still sends an `APIResonse`